sections/semantics-embedded-content.include

```diff
@@ -460,10 +460,10 @@ was an English &lt;a href="/wiki/Music_hall"&gt;music hall&lt;/a&gt; singer, ...
     </dd>
   </dl>
 
-  The authoring requirements in this section only apply if the <{source}> element has
+  The authoring requirements in this section only apply if the <a element lt="picture source"><code>source</code></a> element has
```

Comment:
I tried that but then I get errors like
"
FATAL ERROR: No 'element' refs found for ' elements, followed by one img element, optionally intermixed with '.
"